mojave/structures/decorative.dm

```diff
@@ -32,25 +32,25 @@
 
 //signs/flags//
 
-/obj/structure/fluff/ms13
+/obj/structure/ms13
 	name = "fluff ms13 basetype"
```

Comment:
Density, anchored, and opacity are all defined on the base fluff level but not redefined here. Anchored for sure we'll want to redefine as the default is false.